samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/servlet/event/ExportEventLogServlet.java

```diff
@@ -89,30 +96,37 @@ public void doGet(HttpServletRequest req, HttpServletResponse res) throws Servle
         }
 
         // Prepare data for csv
-        List<String[]> lines = new LinkedList<>() {{
-            // Add headers
-            add(new String[] {
-                RESOURCE_BUNDLE.getString("title"),
-                RESOURCE_BUNDLE.getString("id"),
-                RESOURCE_BUNDLE.getString("user_id"),
-                RESOURCE_BUNDLE.getString("date_startd"),
-                RESOURCE_BUNDLE.getString("date_submitted"),
-                RESOURCE_BUNDLE.getString("duration")
-            });
+        List<String[]> lines = new LinkedList<>();
+
+        List<String> headerList = new ArrayList<>(){{
+                add(RESOURCE_BUNDLE.getString("title"));
+                add(RESOURCE_BUNDLE.getString("id"));
+                add(RESOURCE_BUNDLE.getString("user_id"));
+                add(RESOURCE_BUNDLE.getString("date_startd"));
+                add(RESOURCE_BUNDLE.getString("date_submitted"));
+                add(RESOURCE_BUNDLE.getString("duration"));
+                add(RESOURCE_BUNDLE.getString("errors"));
+                add(RESOURCE_BUNDLE.getString("ipAddress"));
```

Comment:
The IP address is always appended independently of the property, am I right?

Reply:
Correct, good catch! It's fixed